safe_control_gym/controllers/mpc/linear_mpc.py

```diff
@@ -69,6 +73,8 @@ def __init__(
             # prior_info=prior_info,
             output_dir=output_dir,
             additional_constraints=additional_constraints,
+            use_lqr_gain_and_terminal_cost=use_lqr_gain_and_terminal_cost,
+            compute_initial_guess_method='lqr',  # use lqr initial guess by default
```

Comment:
Why would this not be set by the parameters inherited from the MPC class? It should not always be LQR rt, it should be a parameter that can be passed in like all the others? If there is a default, it needs to be set on the MPC init func rt?

Reply:
This is a good point. Right now, for the initial guess computation, I have only two options: lqr or using ipopt. The ipopt option will solve a nonlinear MPC step. I think using a nonlinear MPC solution to warm-start a linear MPC optimization makes less sense, but certainly, we can do this. However, what I have here is actually problematic in the sense that it will always start with an lqr initial guess. I will make it a parameter instead of hard-code it like this.